frontend/src/App.js

```diff
@@ -2,20 +2,33 @@ import './App.css';
 import Header from './Components/Header'
 import EventsRail from './Components/EventsRail';
 import EventTile from './Components/EventTile'
-import { BrowserRouter } from 'react-router-dom';
-import RegistrationForm from './Components/RegistrationForm';
+import {BrowserRouter, Routes,Route} from 'react-router-dom';
+import LoginPage from './Components/LoginRegPage';
+import CreateEventForm from './Components/CreateEventForm';
 function App() {
+
   return (
     <BrowserRouter>
     <div className="App">
-      <Header/>
-      <EventsRail>
-        <EventTile title ="Football 5-a-side" image = "e619f4e1-c156-419e-8d26-f2cc46235e76.jpg" tags = {["5-a-side","SkyFootball"]}/>
-        <EventTile title ="Tennis Doubles" image = "download.jpeg" tags = {["5-a-side","SkyFootball"]}/>
-        <EventTile title ="Running" image = "download (1).jpeg" tags = {["5-a-side","SkyFootball"]}/>
-        <EventTile title ="Football 5-a-side" image = "e619f4e1-c156-419e-8d26-f2cc46235e76.jpg" tags = {["5-a-side","SkyFootball"]}/>
-        <EventTile title ="Tennis Doubles" image = "download.jpeg" tags = {["5-a-side","SkyFootball"]}/>
-      </EventsRail>
+      <Routes>
+        <Route element = {<Header/>}>
+          <Route path="/" element={
+            <EventsRail>
+            <EventTile title ="Football 5-a-side" image = "e619f4e1-c156-419e-8d26-f2cc46235e76.jpg" tags = {["5-a-side","SkyFootball"]}/>
+            <EventTile title ="Tennis Doubles" image = "download.jpeg" tags = {["5-a-side","SkyFootball"]}/>
+            <EventTile title ="Running" image = "download (1).jpeg" tags = {["5-a-side","SkyFootball"]}/>
+            <EventTile title ="Football 5-a-side" image = "e619f4e1-c156-419e-8d26-f2cc46235e76.jpg" tags = {["5-a-side","SkyFootball"]}/>
+            <EventTile title ="Tennis Doubles" image = "download.jpeg" tags = {["5-a-side","SkyFootball"]}/>
+          </EventsRail>
+          }/>
+          <Route path="/MyEvents" element={<h1>My Events Page Goes here</h1>}/>
```

Comment:
With the path naming convention do we go for Capital letters or should we do camel casing, as I think that's what most of the websites are using?

Reply:
Its consistent throughput the website so I don't think it matters much but can change it if that's what you want.

Reply:
yeah I don't think it matters that much but just wanted to make sure 😄